css/styles.css

```diff
@@ -73,3 +73,8 @@ ul.no-bullets {
   text-decoration: none;
   font-size: larger;
 }
+
+.margin-float {
+  margin-top: 1.4em;
+  float: right;
```

Comment:
You should try and avoid `float`. You should utilize `flex` & `justify-content` to move things over to the right

Reply:
Thanks Shawn. Updated renderEntry function to use flex and justify-content instead of float.